src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
-    public int getUserScore(String[] records, String email) {
-        return 0;
+    public int getUserScore(String[] records, String emailInput) throws UserNotFoundException {
```

Comment:
should we throw an exception in the method signature?

Reply:
done